### PR TITLE
[bitnami/common] feat: add redis passwords validations

### PR DESCRIPTION
**Pull request by dani8art**
Signed-off-by: darteaga <darteaga@vmware.com>

**Description of the change**

Add validations for redis passwords when upgrading the chart

**Benefits**

We will upgrade redis and redis dependant charts without missing passwords

**Possible drawbacks**

N/A

**Applicable issues**

N/A

**Additional information**

rel #4069

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: bitnami/common/templates/validations/_redis.tpl [hunk 1]
```diff
@@ -0,0 +1,72 @@
+
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Validate Redis required passwords are not empty.
+
+Usage:
+{{ include "common.validations.values.redis.passwords" (dict "secret" "secretName" "subchart" false "context" $) }}
+Params:
+  - secret - String - Required. Name of the secret where postgresql values are stored, e.g: "redis-passwords-secret"
```
Comment (juan131) [inline bitnami/common/templates/validations/_redis.tpl:9]: Typo `postgresql` -> `redis`

File: bitnami/common/templates/validations/_redis.tpl [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  - subchart - Boolean - Optional. Whether postgresql is used as subchart or not. Default: false
```
Comment (juan131) [inline bitnami/common/templates/validations/_redis.tpl:10]: Same typo here

File: bitnami/common/templates/validations/_redis.tpl [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+{{/*
+Auxiliar function to get the right prefix path for the values
+
+Usage:
+{{ include "common.redis.values.key.prefix" (dict "subchart" "true" "context" $) }}
+Params:
+  - subchart - Boolean - Optional. Whether redis is used as subchart or not. Default: false
+*/}}
+{{- define "common.redis.values.keys.prefix" -}}
+  {{- if .subchart -}} redis. {{- else -}}{{- end -}}
```
Comment (juan131) [inline bitnami/common/templates/validations/_redis.tpl:71]: Are the spaces between `}} redis. {{` necessary or problematic?

**Review by juan131 — APPROVED** [review]
Please check the minor comments before landing

File: bitnami/common/templates/validations/_redis.tpl (hunk 3 above)
Comment (dani8art) [reply bitnami/common/templates/validations/_redis.tpl:71]: Having the `-` it shouldn't be problematic but I agree, removing them just in case ;)

**Pull request merged**